model.cx

```diff
@@ -7,6 +7,11 @@ struct Result:/
 Result get(string url, string fields)
 Result post(string url, string fields)
 string encodeFields(string fields)
+int16 set_user(string user)
+string get_user()
+int16 set_password(string password)
+string get_password()
+int32 getTimeout()
```

Comment:
Inconsistency in naming: suggest to change to `get_timeout`

Reply:
Out of scope of this PR. Plan to follow-up with new PR.

Reply:
Done.